netboot-oci-artifacts.md

```diff
@@ -0,0 +1,96 @@
+# Packaging Network Boot files
+
+Netboot files (i.e. PXE) can be packaged as [OCI Artifacts](https://github.com/opencontainers/image-spec/blob/main/artifacts-guidance.md)
+using an [OCI image manifest](https://github.com/opencontainers/image-spec/blob/master/manifest.md)
+and stored in OCI registry.
+
+## Image config object
+
+The configuration object must have the MIME type `application/vnd.oci.empty.v1+json`, and must be
+an empty JSON object ('{}').
+See the [Guidance for an empty descriptor](https://github.com/opencontainers/image-spec/blob/main/manifest.md#guidance-for-an-empty-descriptor) 
+
+Because the `config.mediaType` is set to the empty value, the `artifactType` MUST be defined and
+currently it has MIME type `application/vnd.unknown.artifact.v1`.
+
+## Image layers
+
+Each layer in the manifest represents provided at build time PXE files:
+
+* vmlinuz
+* initrd.img
+* shim.efi
+* grubx64.efi
+
+Each layer must have:
+
+* the MIME type `application/x-netboot-file` or `application/x-netboot-file+zstd` if compressed
+* the annotations `org.opencontainers.image.description` and `org.opencontainers.image.title`
+  describing each file
+
+If the layer is not stored in a registry, then the optional `urls` property of the layer descriptor
+can be used to point to the layer contents.
+
+
+For the rest of image definition refer to the OCI specs.
+
+
+## Example OCI Image Manifest containing PXE files stored as OCI artifacts
```

Comment:
Did not add any info related to building itself - like steps or tools to use. Felt like outside of the scope for the specs but could be added at the bottom.